crates/primitives/src/chain/spec.rs

```diff
@@ -287,6 +287,7 @@ pub static OP_MAINNET: Lazy<Arc<ChainSpec>> = Lazy::new(|| {
             (Hardfork::Bedrock, ForkCondition::Block(105235063)),
             (Hardfork::Regolith, ForkCondition::Timestamp(0)),
             (Hardfork::Canyon, ForkCondition::Timestamp(1704992401)),
+            (Hardfork::Delta, ForkCondition::Timestamp(1708560000)),
```

Comment:
where did you find this? Is there an op mainnet genesis link somewhere?

Reply:
Added comment with link 
https://github.com/paradigmxyz/reth/pull/8114/commits/2ea4bccbb4f04bebd337fbc46371030a393115df

Reply:
ah, reading this it looks like that's an op-node (CL) change, so it does not need to exist here

Reply:
Where do you see that?

Reply:
see https://docs.optimism.io/builders/node-operators/network-upgrades

> The Delta upgrade uses a L2 block-timestamp activation-rule, and is specified only in the rollup-node (delta_time).